test/SIMD.bool8x16.asmjs/rlexe.xml

```diff
@@ -23,6 +23,13 @@
       <compile-flags>-bgjit- -simdjs -simd128typespec -asmjs- -off:simplejit -mic:1</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>testConstructor.js</files>
+      <tags>exclude_dynapogo,exclude_ship</tags>
+      <compile-flags>-asmjs -simdjs -off:simplejit -maic:0 -mic:0</compile-flags>
```

Comment:
Can you fix the missing flags here to be:
<compile-flags>-bgjit- -on:asmjs -testtrace:asmjs -simdjs -AsmJsStopOnError -maic:0</compile-flags>
And also add this test:
<compile-flags>-bgjit- -on:asmjs -testtrace:asmjs -simdjs -AsmJsStopOnError -lic:1</compile-flags>